tests/framework/console/controllers/MigrateControllerTest.php

```
@@ -442,16 +442,21 @@ public function testNamedMigrationWithCustomLimit(): void
 
     public function testCreateLongNamedMigration(): void
     {
-        $this->setOutputCallback(fn($output) => null);
-
         $migrationName = str_repeat('a', 180);
 
         $this->expectException('yii\console\Exception');
         $this->expectExceptionMessage('The migration name is too long.');
 
         $controller = $this->createMigrateController([]);
         $params[0] = $migrationName;
```

PHPMD (2.15.0)

[warning] Avoid using undefined variables such as '$params' which will lead to PHP notices. (undefined)

(UndefinedVariable)